src/foam/core/ProbeAgency.java

```diff
@@ -0,0 +1,12 @@
+package foam.core;
+
+public class ProbeAgency
```

Comment:
Don't you still want to add the agents so you know what you would have done?
Just extend CompoundContextAgent and then NOP the execute method.